core/lib/Thelia/Config/I18n/en_US.php

```diff
@@ -55,7 +55,7 @@
     'Associate documents' => 'Associate documents',
     'Associate images' => 'Associate images',
     'At least one of cart products categories is %op% <strong>%categories_list%</strong>' => 'At least one of cart products categories is %op% <strong>%categories_list%</strong>',
-    'Attribut' => 'Attribut',
+    'Attribut' => 'Attribute',
```

Comment:
I don't agree, the key can't change. It doesn't matter if there is a typo, you can fixed it in the translation.